index.js

```diff
@@ -8,11 +8,16 @@ var AWS = require('aws-sdk-promise');
 var moment = require('moment');
 var readFileAsync = Bluebird.promisify(fs.readFile);
 var options = {};
+var util = require('util');
+var BaseStore = require(path.join(process.cwd(), 'core/server/storage/base'));
```

Comment:
If you run ghost as a dependency then `BaseStore` has to be loaded from the `ghost` module. To be compatible with both variations, I would do this:

```javascript
var BaseStore;
try {
    BaseStore = require('ghost/core/server/storage/base');
} catch (e) {
    if (e.code !== 'MODULE_NOT_FOUND') throw e;
    BaseStore = require(path.join(process.cwd(), 'core/server/storage/base'));
}
```